main.js

```diff
@@ -106,9 +147,13 @@ const sortFunctions = [
     selection,
 ];
 
-sortFunctions.forEach(sortFunction => {
-    console.log(`testing performance of ${sortFunction.name}`)
-    inputSizes.forEach(inputSize => {
-        benchmarkSort(sortFunction, inputSize, maxNumber);
-    });
-});
\ No newline at end of file
+// sortFunctions.forEach(sortFunction => {
+//     console.log(`testing performance of ${sortFunction.name}`)
+//     inputSizes.forEach(inputSize => {
+//         benchmarkSort(sortFunction, inputSize, maxNumber);
+//     });
+// });
+
+let test = [4, 2, 8, 9, 3];

```

Comment:
remove this test and re-instate the benchmarks with mergesort and quicksort included